### PR TITLE
Add incentive model

### PR DESCRIPTION
**Pull request by Tommo-L**
Close https://github.com/neo-project/neo/issues/1617

We have imported the concept of `Epoch`, At the beginning of epoch, the economic parameters and committees decide the economic model during the epoch.

### CODE REVIEW

File: src/neo/ProtocolSettings.cs [hunk 1]
```diff
@@ -91,6 +92,7 @@ private ProtocolSettings(IConfigurationSection section)
                     "seed5.neo.org:10333"
                 };
             this.MillisecondsPerBlock = section.GetValue("MillisecondsPerBlock", 15000u);
+            this.Epoch = section.GetValue("Epoch", 5760u);
```
Comment (shargon) [inline src/neo/ProtocolSettings.cs:95]:
```suggestion
            this.Epoch = section.GetValue("Epoch", 5760u); // 24 hours
```

File: src/neo/SmartContract/Native/Tokens/EconomicParameter.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+namespace Neo.SmartContract.Native.Tokens
+{
+    public class EconomicParameter : IInteroperable
+    {
+        public BigInteger GasPerBlock;
+        public uint NeoHoldersRewardRatio;
+        public uint CommitteesRewardRatio;
+        public uint VotersRewardRatio;
+
+        public uint TotalRewardRatio => NeoHoldersRewardRatio + CommitteesRewardRatio + VotersRewardRatio;
```
Comment (shargon) [inline src/neo/SmartContract/Native/Tokens/EconomicParameter.cs:15]: It could produce a overflow? can we change the variables to byte?

**Comment by shargon** [comment]
Conflicts

**Comment by Tommo-L** [comment]
I'll close this pr, and send another pr.

**Pull request closed without merging**